patches/server/0807-Allow-explosions-to-affect-projectiles.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: c0repwn3r <coredoescode@gmail.com>
+Date: Fri, 20 Aug 2021 14:59:56 -0400
+Subject: [PATCH] Allow explosions to affect projectiles
+
+
+diff --git a/src/main/java/net/minecraft/world/level/Explosion.java b/src/main/java/net/minecraft/world/level/Explosion.java
+index 132140e00865fcf84ebe03ffcbc2f30ac11a0b35..2b546b4661d6f3aba36e19d5538a1546907e4478 100644
+--- a/src/main/java/net/minecraft/world/level/Explosion.java
++++ b/src/main/java/net/minecraft/world/level/Explosion.java
+@@ -249,7 +249,8 @@ public class Explosion {
+                         entity.forceExplosionKnockback = false;
+                         boolean wasDamaged = entity.hurt(this.getDamageSource(), (float) ((int) ((d13 * d13 + d13) / 2.0D * 7.0D * (double) f2 + 1.0D)));
+                         CraftEventFactory.entityDamage = null;
+-                        if (!wasDamaged && !(entity instanceof PrimedTnt || entity instanceof FallingBlockEntity) && !entity.forceExplosionKnockback) {
++			// BandagedPaper - Allow tnt to move projectiles
```

Comment:
1) This comment should be in the same line as the if statement, reducing the diff to a single line of change.
2) `// Paper`

Reply:
Will move the comment and change it, as stated I just copied the patch from Bandaged

Reply:
Comment moved and renamed.